### PR TITLE
Add some explanation for tiflash tuning performance doc

### PR DESCRIPTION
**Pull request by CrystalAnalyst**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Add some explanation for tiflash tuning performance doc

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [ ] v7.6 (TiDB 7.6 versions)
- [x] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/15496
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by dveeden** [comment]
From the linter:
```
tiflash/tune-tiflash-performance.md: 310: MD012/no-multiple-blanks Multiple consecutive blank lines [Expected: 1; Actual: 2]
```

File: tiflash/tune-tiflash-performance.md
Comment (dveeden) [inline tiflash/tune-tiflash-performance.md:309]:
```suggestion
```

To fix: `tiflash/tune-tiflash-performance.md: 310: MD012/no-multiple-blanks Multiple consecutive blank lines [Expected: 1; Actual: 2]`

File: tiflash/tune-tiflash-performance.md
Comment (dveeden) [inline tiflash/tune-tiflash-performance.md:308]:
```suggestion
The following example shows the query result before and after `tidb_max_tiflash_threads` is re-configured. Before `tidb_max_tiflash_threads` is set, the concurrency of request execution for a single TiFlash instance is 8 threads. The cluster has a total of 3 TiFlash instances, so the total number of threads executed by requests on all TiFlash instances is 24 (8 × 3). When `tidb_max_tiflash_threads` is set to `20`, the total number of threads executed by requests on all TiFlash instances is 60 (20 × 3).
```

File: tiflash/tune-tiflash-performance.md
Comment (CrystalAnalyst) [reply tiflash/tune-tiflash-performance.md]: done.

File: tiflash/tune-tiflash-performance.md
Comment (CrystalAnalyst) [reply tiflash/tune-tiflash-performance.md]: thanks for your suggestion! I've appended them.

**Review by dveeden — APPROVED** [review]

**Comment by rpaik** [comment]
Thanks for your contribution to TiDB Docs Dash @CrystalAnalyst 🎊

**Comment by CrystalAnalyst** [comment]
@Oreoxmt @qiancai Hi ~ would you take some time review my changes ? I'd appreciated that, thanks!

**Comment by CrystalAnalyst** [comment]
@Oreoxmt Sorry to bother, but the DDL is approaching, can you review  and approve the change ?  If there's still room for improvement, I'm glad to change.

**Comment by Oreoxmt** [comment]
@CrystalAnalyst 

Thank you for reaching out, and no worries at all. Just to clarify, the deadline for this TiDB Docs Dash event is to merge this PR before January 26th (see https://www.pingcap.com/event/tidb-docs-dash/). Currently, we're handling a significant backlog of PRs waiting for review. I'll prioritize reviewing and merging your PR, but it might take a bit of time due to the high volume of pending requests.

**Comment by CrystalAnalyst** [comment]
> @CrystalAnalyst
> 
> Thank you for reaching out, and no worries at all. Just to clarify, the deadline for this TiDB Docs Dash event is to merge this PR before January 26th (see https://www.pingcap.com/event/tidb-docs-dash/). Currently, we're handling a significant backlog of PRs waiting for review. I'll prioritize reviewing and merging your PR, but it might take a bit of time due to the high volume of pending requests.

Oh! I see. Wish you good luck!

File: tiflash/tune-tiflash-performance.md
Comment (Oreoxmt) [inline tiflash/tune-tiflash-performance.md:308]:
```suggestion
The following example shows the query result before and after `tidb_max_tiflash_threads` is re-configured. Before `tidb_max_tiflash_threads` is set, the concurrency of request execution for a single TiFlash instance is 8 threads. Since the cluster has a total of 3 TiFlash instances, the total number of threads executed by requests on all TiFlash instances is 24 (8 × 3). After `tidb_max_tiflash_threads` is set to `20`, the total number of threads executed by requests on all TiFlash instances is 60 (20 × 3).
```

**Review by Oreoxmt — APPROVED** [review]

File: tiflash/tune-tiflash-performance.md
Comment (ran-huang) [inline tiflash/tune-tiflash-performance.md:308]:
```suggestion
The following example shows the query result before and after `tidb_max_tiflash_threads` is re-configured. Before `tidb_max_tiflash_threads` is set, the concurrency of request execution for a single TiFlash instance is 8 threads. Since the cluster has a total of 3 TiFlash instances, the total number of threads for request execution on all TiFlash instances is 24 (8 × 3). After `tidb_max_tiflash_threads` is set to `20`, the total number of threads for request execution on all TiFlash instances is 60 (20 × 3).
```

**Pull request merged**
